oauth_provider/runtests/urls.py

```diff
@@ -13,9 +13,9 @@ def resource_None_scope_view(request):
     return HttpResponse()
 
 
-urlpatterns = patterns('',
```

Comment:
The `tox` tests pass for all versions of Django, so I assume it should all be good in that case? It also struck me as odd, but I imagine that it should have been caught otherwise?